extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -244,6 +262,11 @@ export class ConsentUI {
       this.baseInstance_.getViewport().removeFromFixedLayer(this.parent_);
       toggle(dev().assertElement(this.ui_), false);
       this.isVisible_ = false;
+
+      if (this.elementWithFocusBeforeShowing_) {
```

Comment:
As we agreed, if `this.elementWithFocusBeforeShowing_ == null`, can we focus on the first element of the page.

Reply:
Done. Used `children[0]` instead of `firstChild` due to: https://stackoverflow.com/questions/10474676/get-immediate-first-child-element

Reply:
Thank you. Do you know what happens if `children[0]` is not focusable? Ideally we want to find and cache the first focusable element. Don't want to block you, we can leave that as a TODO